packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,9 @@
+{
+  "pci_databases_service_integration_tab_title": "Intégration de services",
+  "pci_databases_service_integration_tab_descritpion": "L’intégration de services vous permet de connecter différentes solutions Kafka.",
+  "pci_databases_service_integration_tab_kafka_service": "Service Kafka",
+  "pci_databases_service_integration_tab_add_kafka": "Ajouter un Kafka",
+  "pci_databases_service_integration_tab_no_kafka_service": "Vous ne pouvez pas ajouter un Kafka car vous n'avez plus d'autre service Kafka disponible à connecter.",
+  "pci_databases_service_integration_tab_unknown_service": "Service non trouvé",
+  "pci_databases_service_integration_tab_service_ready": "L'intégration de votre service Kafka est prête"
```

Comment:
```suggestion
  "pci_databases_service_integration_tab_service_ready": "L'intégration de votre service Kafka est prêt"
```

Reply:
same: I cannot change the text as it has been already validated by content team. Here it is the integration that is ready, so the text is ok